src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,58 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Comments } from '../../types/Comments';
+import { User } from '../../types/User';
```

Comment:
you still didn't add it
type in terminal `git add .`, then commit and push
`git add .` will add newly created files